libs/api/domains/cms/src/lib/models/districts.model.ts

```diff
@@ -0,0 +1,36 @@
+import { Field, ID, ObjectType } from '@nestjs/graphql'
+
+import { IDistricts } from '../generated/contentfulTypes'
+import { Image, mapImage } from './image.model'
+import { Link, mapLink } from './link.model'
+import { SystemMetadata } from 'api-cms-domain'
+
+@ObjectType()
+export class Districts {
+  @Field(() => ID)
+  id: string
+
+  @Field()
+  title: string
+
+  @Field({ nullable: true })
+  description: string
+
+  @Field({ nullable: true })
+  image?: Image
+
+  @Field(() => [Link])
+  links?: Array<Link>
+}
+
+export const mapDistricts = ({
+  sys,
+  fields,
+}: IDistricts): SystemMetadata<Districts> => ({
+  typename: 'Districts',
+  id: sys.id,
+  title: fields.title,
+  description: fields.description,
+  image: mapImage(fields.image),
+  links: fields.links.map(mapLink),
```

Comment:
We should provide defaults for all non optional fields to support preview since it can have empty fields